backend/src/routes/getOtp.ts

```diff
@@ -19,26 +20,43 @@ export function getOtp(fastify: FastifyInstance) {
       schema: {
         body: {
           type: 'object',
-          required: ['phone_number'],
+          required: ['phone_number', 'nickname'],
           properties: {
             phone_number: { type: 'string', pattern: '^\\+[1-9]\\d{1,14}$' },
+            nickname: { type: 'string', pattern: '^[A-Za-z]{1,20}$' },
           },
         },
       },
     },
     async (request, reply) => {
       try {
-        const { phone_number } = request.body as { phone_number: string };
+        const { phone_number, nickname } = request.body;
 
         // validating if phone number exists in db
-        const record_phone_number = await fastify.db
+        const record_by_phone_number = await fastify.db
           .select()
           .from(registration)
           .where(eq(registration.phone_number, phone_number))
           .orderBy(desc(registration.created_at));
 
-        if (!record_phone_number.length) {
-          return reply.code(400).send({ message: 'No record exists with current phone number' });
+        if (!record_by_phone_number.length) {
+          try {
+            await fastify.db.insert(schema.registration).values({
+              phone_number,
+              nickname,
+            });
+
+            return reply.code(200).send(true);
```

Comment:
```suggestion
            return reply.code(200).send({ ok: true });
```
To be consistent with other APIs